.pre-commit-config.yaml

```diff
@@ -53,3 +51,10 @@ repos:
         entry: .scripts/verify_crate_versions.sh
         stages: [pre-commit, pre-merge-commit, manual]
         pass_filenames: false
+
+      - id: rustfmt
+        name: rustfmt
+        language: system
+        entry: cargo +nightly-2025-01-15 fmt --all -- --check
```

Comment:
```suggestion
	# Pinning to a specific rustc version, so that we get consistent formatting
        entry: cargo +nightly-2025-01-15 fmt --all -- --check
```